app/workers/web_mentions/send_web_mention.rb

```diff
@@ -0,0 +1,17 @@
+module WebMentions
+  class SendWebMention
+    include Sidekiq::Worker
+    sidekiq_options queue: :low_priority, retry: 15, lock: :until_executing
+
+    def perform(comment_id)
+      comment = Comment.find_by(id: comment_id)
+      return unless comment
+
+      return unless comment.commentable.support_webmentions
```

Comment:
We do keep comments even when commentable's have been deleted so we need a safety operator here. 
```ruby
comment.commentable&.support_webmentions
```

Reply:
Yes, and since we are using it several times, it may be easier to do something like

```ruby
commentable = Comment.find_by(id: comment_id)&.commentable
return unless commentable
return uless commentable.support_webmentions
```

which also protects the other uses of `commentable` in this method.